### PR TITLE
move to lhapdf6 6.1.4 and make a few fixes

### PR DESCRIPTION
**Pull request by mkirsano**

### CODE REVIEW

**Comment by bendavid** [comment]
@mkirsano,
I see an explicit list of pdfs in
https://github.com/cms-sw/cmsdist/pull/945/files#diff-3aca08d05fafad4c82458388862b75c7L4

Does this need to be updated for the additional pdfs available in 613/614? (in particular nnpdf3.0)

**Comment by mkirsano** [comment]
You are right, Josh. I also added some automatization.

**Comment by bendavid** [comment]
Thanks,
If  lhapdf6_makeLinks.file is now generated automatically, does there still need to be a copy in the repository?

**Comment by mkirsano** [comment]
It is to be generated semiautomatically when lhapdf6 version is upgraded, on a computer with cvmfs. It still needs to be in the repository.

**Comment by bendavid** [comment]
+1

**Comment by bendavid** [comment]
Can this one be moved forward?
(We may need to backport this to 53x on short notice, so having it in 72x ASAP would be helpful)

File: lhapdf6_makeScript.sh [hunk 1]
```diff
@@ -0,0 +1,26 @@
+#TODO: check hat versions coincide
+
+rm -f lhapdf6_makeLinks.file
```
Comment (ktf) [inline lhapdf6_makeScript.sh:3]: Can you elaborate on what is happening here? Is this generating the lhapdf6_makeLinks.file file on the fly?

Comment (bendavid) [inline lhapdf6_makeScript.sh:3]: My understanding is that this script is run manually when preparing a new pull request.  Then the version of lhapdf6_makeLinks which is actually used finally is the one in the repository/pull request.

Comment (ktf) [inline lhapdf6_makeScript.sh:3]: Yeah, I now see it's listing /cvmfs… sigh… I personally do not like the idea of having in CMSDIST a random script running on a random machine using an hardcoded path, but I guess I'll live with it.. @degano can you test this out and merge it?

**Comment by ghost** [comment]
+1
Tested compilation. Merging.

**Pull request merged**